src/OrbitGl/Button.cpp

```diff
@@ -107,23 +119,26 @@ void Button::DoDraw(PrimitiveAssembler& primitive_assembler, TextRenderer& /*tex
 
 void Button::DrawSymbol(PrimitiveAssembler& primitive_assembler) {
   const Color kSymbolColor(191, 191, 192, 255);
+  const Color kHighlightedSymbolColor(255, 255, 255, 255);
```

Comment:
Are the `kBaseColor` and `kHighlightColor` used in some other place? If not, then I would prefer to define them in `DoDraw`.

(Very personal preference XD, feel free to ignore) You are naming those colors in different way: `kBaseColor` vs `kHighlightColor`, and `kSymbolColor` vs `kHighlightedSymbolColor`. Maybe change to use `kSymbolBaseColor` and `kSymbolHighlightColor` here.

Reply:
> Are the kBaseColor and kHighlightColor used in some other place? If not, then I would prefer to define them in DoDraw.
The answer is not. I'd guess Florian has a reason for that (like being able to quickly change them without entering to the functions). But it's not consistent with what I did, so I will move them.

> (Very personal preference XD, feel free to ignore) You are naming those colors in different way: kBaseColor vs kHighlightColor, and kSymbolColor vs kHighlightedSymbolColor. Maybe change to use kSymbolBaseColor and kSymbolHighlightColor here.

I'm not sure if I understand. The base color and highlight color are not related with the +/- symbol. Why I would rename it including symbol in the name?

Reply:
Thanks for the offline discussion!